src/components/album/AlbumPreview.js

```diff
@@ -1,9 +1,12 @@
 import React, { useEffect, useState } from 'react';
 import { CardContent } from '@mui/material';
-
+import axiosInstance from '../../apis/axiosClient';
+import WriteReview from '../review/WriteReview';
 const AlbumPreview = ({
+  albumId,
   spotifyUrl,
   style = {},
+  contentLayout = 'column',
```

Comment:
Since we've made this by default `"column"`. We should make sure we don't break how the `localhost:3000/chat` page was coded, where it wanted the album preview to be side by side (content layout: "row") with the chat box (or check with @mvstoyan  if we're fine with the design of that page changing to a column layout). 

If we want to make sure the chat page continues to be rendered in horizontal fashion, then we should update `AlbumChat.js` to pass in `contentLayout="row"` to the `<AlbumPreview>` component that it renders.

Reply:
@akosasante sure, my bad, will leave it in a horizontal way